client/src/homePage/FormStructure.svelte

```diff
@@ -59,6 +67,13 @@
           data = await shrinkUrlService( token, longUrl );
           if(!data.linkId){
             error = 'The URL you entered is not valid. Please refresh and try again with a valid URL.';
+            toast.push('Invalid URL',{
+        theme:{
+          '--themeBackground':'cyan',
+          '--toastColor':'white',
```

Comment:
I will remove the "un-applied"  properties `'--themeBackground':'cyan' and '--toastColor':'white'` and change the border-radius but it would be helpful if you could specify the hex code of the red color that I should use ??

Reply:
@shawshankkumar 
update on the French toast situation :- 
 - added the toast objects to constants.js as  toastSuccess and toastFail
 - changed the color of fail toast from  #FF0000 (usual red) to F0634D (theme color)
 - added toast.pop() on top of toast.push("Link copied" , toastSuccess) to eliminte a bug (where user keeps clicking on copy button to resulting in infinite toasts notifications